src/p5.dance.js

```diff
@@ -1138,6 +1157,27 @@ module.exports = class DanceParty {
       title,
     };
 
+    let currentMeasure = Math.floor(this.getCurrentMeasure());
+    this.sprites_.forEach(sprite => {
+      if (sprite.alternatingMoveInfo) {
+        let alternatingMoveInfo = sprite.alternatingMoveInfo;
+        let val = Math.floor((currentMeasure - sprite.alternatingMoveInfo.start) / sprite.alternatingMoveInfo.cadence);
```

Comment:
Could we ever end up with an off-by-one error? i.e. I'm imagining this scenario:
val = 1
val = 2
val = 3
val = 3
val = 5

Reply:
Synced offline- this is not possible. `start` and `cadence` never change, and `currentMeasure` increases at a constant rate, so the quotient will also increase at a constant rate.